azure-devops/azext_devops/test/repos/test_pull_request.py

```diff
@@ -134,6 +134,38 @@ def test_create_pull_request(self):
         assert pr_object_from_create_call.target_ref_name == resolve_git_ref_heads(self._TEST_TARGET_BRANCH)
         assert pr_object_from_create_call.work_item_refs == None
 
+
+    def test_create_pull_request_with_duplicate_reviwer(self):
+
+        test_pr_id = 1
+
+        # set return values
+        self.mock_create_PR.return_value.id = test_pr_id
+        self.mock_resolve_reviewers_as_ids.return_value = ['id1']
+
+        response = create_pull_request(project = self._TEST_PROJECT_NAME,
+        repository = self._TEST_REPOSITORY_NAME,
+        source_branch = self._TEST_SOURCE_BRANCH,
+        target_branch = self._TEST_TARGET_BRANCH,
+        title = self._TEST_PR_TITLE,
+        description = self._TEST_PR_DESCRIPTION,    
+        reviewers = ['a@b.com','A@b.com'],
```

Comment:
we don't have an assert for this ?

Reply:
@roshan-sy : we can evaluate if https://github.com/Azure/azure-devops-cli-extension/blob/c5df216e195ddd6fc085298abd6bb39bdc868ca5/azure-devops/azext_devops/test/repos/test_pull_request.py#L90 is called only with one reviewer

Reply:
![image](https://user-images.githubusercontent.com/83656375/134617754-0781795e-f9f6-4792-8dcf-304a36664aa0.png)